src/commands/partial visibality question/private.ts

```diff
@@ -31,6 +31,9 @@ export default class extends Command {
 		}
 
 		privThread.guild.members.fetch();
+		privThread.guild.members.cache.filter(mem => mem.roles.cache.has(ROLES.STUDENT_ADMIN)).forEach(user => {
```

Comment:
You could probably do this in one pass of the guild roles just by adding a `|| mem.roles.cache.has(ROLES.STUDENT_ADMIN)` to the previously existing filteron line 37